WeakAuras/Transmission.lua

```diff
@@ -1064,7 +1075,13 @@ local function scamCheck(codes, data)
   end
 end
 
-local ignoredForDiffChecking = WeakAuras.internal_fields
+local ignoredForDiffChecking = {}
```

Comment:
There's actual a Utility function in Util.lua that can do those loops in one line:

local ignoredForDiffChecking = Mixin({}, WeakAuras.internal_fields, WeakAuras.non_transmissable_fields)

Reply:
Right, I forgot that exists.